common/formats/cue.cpp

```diff
@@ -297,15 +300,30 @@ void CueSheet::parseTracksContext(const char *line) {
 	} else if (command == "INDEX") {
 		int indexNum = atoi(nexttok(s, &s).c_str());
 		int frames = parseMSF(nexttok(s, &s).c_str());
+		// If index 1 is the first specified index,
+		// reuse the last specified pregap
+		if (indexNum > 0 && (int)_tracks[_currentTrack].indices.size() == 0) {
+			// Don't allow index 0, if not specified, to fall below 0
+			int index;
+			if ((frames - lastPregap) < 0) {
+				index = 0;
+			} else {
+				index = frames - lastPregap;
+			}
+			_tracks[_currentTrack].indices.push_back(index);
+		}
 
-		for (int i = (int)_tracks[_currentTrack].indices.size(); i <= indexNum; i++)
+		for (int i = (int)_tracks[_currentTrack].indices.size(); i <= indexNum; i++){
```

Comment:
```suggestion
		for (int i = (int)_tracks[_currentTrack].indices.size(); i <= indexNum; i++) {
```

note space before opening curly brace